Sources/Private/Model/Keyframes/Keyframe.swift

```diff
@@ -14,12 +14,12 @@ import Foundation
  Keyframe represents a point in time and is the container for datatypes.
  Note: This is a parent class and should not be used directly.
```

Comment:
Should this note be reworded now that this is part of the public API? And this type moved out of the private folder?

Reply:
aha good point